mst/connection-pools.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Connection pools
-excerpt: Pool client connections to keep server resource us low
+excerpt: Pool client connections to your TimescaleDB database to use less server resources
```

Comment:
```suggestion
excerpt: Pool client connections to your TimescaleDB database to use fewer server resources
```

Reply:
These would have to be countable, which they are not (had to research this myself)